flatdata-py/generator/tests/generators/test_rust_generator.py

```diff
@@ -29,8 +30,8 @@ def test_constants_are_declared_correctly():
 """,
 """
 // Large constant
-pub const FOOBAR: u64 = 1_000_000_000;
```

Comment:
I am not sure this will work nicely once I switch to expectations as just simple files.